feat: Business trip patch

0xD0M1M0 opened the pull request:
Add Patch to add Business Trip with Expense Claim.

For some reason, the additions in "erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.py" do not work for me. Maybe a hint would be great.

0xD0M1M0 commented:
when running the patch: 

frappe.exceptions.DuplicateEntryError: ('Expense Claim Type', 'Additional meal expenses', IntegrityError(1062, "Duplicate entry 'Additional meal expenses' for key 'PRIMARY'"))

I guess one time effect because I added it manually before. Don't know if anyone else is affected, but can be fixed with a "try"

barredterra commented:
That's weird... I can run the patch again and again without issues. Usually, existing records should not get re-imported because of this line:

https://github.com/alyf-de/erpnext_germany/blob/8a6a4a609d331252c98ba5abb7fdfcf65a647705/erpnext_germany/install.py#L33-L34

0xD0M1M0 commented:
The check looks fine, but migrate throws that error. Renamed my old one, ran migrate, renamed the new one, renamed the old one back and now migrate and the patch don't throw any error.
